### PR TITLE
fix: Fix service export example doc

### PR DESCRIPTION
**Pull request by dsimansk**
## Description

Example of `service export` doesn't contain actual `--mode` parameters.

## Changes

* Fix parameters in the `service export` example

## Reference

Fixes #N/A

### CODE REVIEW

**Review by maximilien — APPROVED** [review]
Good catch and /lgtm

How cool would it be to have a test that scrapes all commands descriptions and try to execute and validate :) them? turns out @cppforlife and I did that in knctl and I believe he kept that going on k14s tools. I never got around adding this here but not impossible.

Let me know if interested and we can add and track and execute this as a follow-up issue.

/ok-to-test

File: pkg/kn/commands/service/export.go [hunk 1]
```diff
@@ -60,9 +60,9 @@ func NewServiceExportCommand(p *commands.KnParams) *cobra.Command {
   # Export a service in JSON format
   kn service export foo -n bar -o json
   # Export a service with revisions
-  kn service export foo --with-revisions --mode=resources -n bar -o json
+  kn service export foo --with-revisions --mode=export -n bar -o json
   # Export services in kubectl friendly format, as a list kind, one service item for each revision
-  kn service export foo --with-revisions --mode=kubernetes -n bar -o json`,
+  kn service export foo --with-revisions --mode=replay -n bar -o json`,
```
Comment (navidshaikh) [inline pkg/kn/commands/service/export.go:65]: should we add a line separator in each example with this to align with other command examples?

Comment (dsimansk) [reply pkg/kn/commands/service/export.go:65]: Sure, will do.

Comment (dsimansk) [reply pkg/kn/commands/service/export.go]: PTAL https://github.com/knative/client/pull/1006/commits/4aaab55d8e7456dcae6809986a9d808f6bf2d87a.

**Review by navidshaikh — APPROVED** [review]
/lgtm
/approve

thanks!

**Pull request merged**